src/components/header/header.module.scss

```
@@ -1,47 +1,74 @@
-.header_top {
-  width: 100%;
-  height: 200px;
-
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-
-  background-color: black;
-
-  background-image: url("./gerasimov.png");
-  background-size: contain;
-  background-repeat: no-repeat;
-  background-position: 53% center;
-}
-
-.phone_blind {
-  margin-left: 50px;
-  margin-top: 20px;
-
-  display: flex;
-  flex-direction: column;
-
-  .phone {
-    font-family: "Mohave";
-    font-style: normal;
-    font-weight: 500;
-    font-size: 25px;
-    line-height: 35px;
-    text-decoration: none;
-
-    margin-bottom: 40px;
-
-    color: #ececec;
-  }
-
-  .blind {
-    font-family: "Montserrat";
-    font-style: normal;
-    font-weight: 400;
-    font-size: 14px;
-    line-height: 18px;
-    text-decoration: none;
-
-    color: #ececec;
-  }
+.header_top {
+  width: 100%;
+  height: 200px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  background-color: black;
+
+  background-image: url("./gerasimov.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: 53% center;
+}
+
+// .phone_blind {
+//   margin-left: 50px;
+//   margin-top: 20px;
+
+//   display: flex;
+//   flex-direction: column;
+
+//   .phone {
+//     font-family: "Mohave";
+//     font-style: normal;
+//     font-weight: 500;
+//     font-size: 25px;
+//     line-height: 35px;
+//     text-decoration: none;
+
+//     margin-bottom: 40px;
+
+//     color: #ececec;
+//   }
+
+//   .blind {
+//     font-family: "Montserrat";
+//     font-style: normal;
+//     font-weight: 400;
+//     font-size: 14px;
+//     line-height: 18px;
+//     text-decoration: none;
+
+//     color: #ececec;
+//   }
+// }
+
+.header_info_gov {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  top: 37px;
+  left: 74px;
+}
+
+.header_info_blind {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  top: 121px;
+  left: 76px;
+}
+
+.header_info_text {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;

```

У макета к сожалению чуть отличается масштаб, поэтому умеешь пожалуйста шрифт
```suggestion
  font-size: 14px;
```